spec/reek/smells/boolean_parameter_spec.rb

```diff
@@ -8,35 +8,35 @@
   context 'parameter defaulted with boolean' do
     context 'in a method' do
       it 'reports a parameter defaulted to true' do
-        src = 'def cc(arga = true) end'
-        expect(src).to smell_of(BooleanParameter, name: 'arga')
+        src = 'def cc(arga = true); arga; end'
+        expect(src).to reek_of(:BooleanParameter, name: 'arga')
       end
       it 'reports a parameter defaulted to false' do
         src = 'def cc(arga = false) end'
-        expect(src).to smell_of(BooleanParameter, name: 'arga')
+        expect(src).to reek_of(BooleanParameter, name: 'arga')
       end
       it 'reports two parameters defaulted to booleans' do
         src = 'def cc(nowt, arga = true, argb = false, &blk) end'
-        expect(src).to smell_of(BooleanParameter,
-                                { name: 'arga' },
-                                { name: 'argb' })
+        expect(src).to reek_of(BooleanParameter,
+                               { name: 'arga' },
+                               { name: 'argb' })
```

Comment:
Here, `reek_of` will combine the two hashes into just one. I'm not sure which value of `name` it will pick :-).